src/ReachSets/project_reach.jl

```diff
@@ -76,21 +74,26 @@ function project_reach(plot_vars::Vector{Int64}, n::Int64,
 
     N = length(Rsets)
 
-    # allocate output
-    RsetsProj = Vector{set_type}(N)
-    oa_arg = set_type == HPolygon ? ɛ : set_type
+    # allocate output and define overapproximation function
+    if approx_sets isa Float64
+        oa = x -> overapproximate(x, HPolygon, approx_sets)
+        RsetsProj = Vector{HPolygon{numeric_type}}(N)
+    else
```

Comment:
`elseif approx_sets isa LazySet` ?

Reply:
From `approx_sets::Union{Float64, Type{<:LazySet}}` this case is the only one remaining, so no need to check that.